src/NServiceBus.NHibernate/NServiceBus.NHibernate.csproj

```diff
@@ -1,30 +1,25 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net452</TargetFramework>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\NServiceBus.snk</AssemblyOriginatorKeyFile>
     <OutputPath>..\..\binaries\</OutputPath>
     <Description>NHibernate integration for NServiceBus</Description>
   </PropertyGroup>
 
   <ItemGroup>
-    <Reference Include="System.Data.DataSetExtensions" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Fody" Version="2.3.23" PrivateAssets="All" />
-    <PackageReference Include="Janitor.Fody" Version="1.5.5" PrivateAssets="All" />
-    <PackageReference Include="Obsolete.Fody" Version="4.3.7" PrivateAssets="All" />
     <PackageReference Include="Particular.CodeRules" Version="0.2.0" PrivateAssets="All" />
     <PackageReference Include="Particular.Packaging" Version="0.1.0" PrivateAssets="All" />
-    <PackageReference Include="NServiceBus" Version="[7.0.0, 8.0.0)" />
-    <PackageReference Include="NHibernate" version="[4.0.4.4000, 5.0.0)" />
-    <PackageReference Include="Iesi.Collections" version="[4.0.2, 5.0.0)" />
+    <PackageReference Include="Fody" Version="3.0.4" PrivateAssets="All" />
+    <PackageReference Include="Janitor.Fody" Version="1.6.1" PrivateAssets="All" />
+    <PackageReference Include="Obsolete.Fody" Version="4.4.0" PrivateAssets="All" />
+    <PackageReference Include="NServiceBus" Version="[7.0.1, 8.0.0)" />
+    <PackageReference Include="NHibernate" version="[5.1.0, 6.0.0)" />
```

Comment:
Any reason to not set the minimum to the latest package version that is out now, 5.1.3? That way users will be getting bug fixes.

Reply:
@bording They should get it according to their "bump up" policy. My reasoning was, that we want to always specify the lowest possible range. Setting `5.1.3` as the lower boundary would make me, as a user, think what's specific about 5.1.3

Reply:
bump up policy?

Reply:
Should we be responsible for making users' NHibernate upgraded to the latest or should it be users' responsiblity? I think it's the latter.

Reply:
I don't think we need to update the package all the time, but I see no reason to release with an older version than the latest that is out when we release, especially given the nature of transitive references. They might not even have a top-level package reference, so they'd be stuck on 5.1.0.